examples/transfer-currency-timing.yaml

```diff
@@ -0,0 +1,120 @@
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - bob: https://bob.example/
+    alice: https://alice.example/
+    banka: https://banka.example/
+    bankb: https://bankb.example/
+    central: https://centralbank.example/
+
+'@id': rgh:valueflows/valueflows/master/examples/transfer-currency-timing.yaml
+'@graph':
+
+  # accounts before
+
+  - '@id': banka:b75d8f6a-e2df-4e52-b36d-1a22a66f4ead
+    '@type': EconomicResource
+    skos:note: Alice's account
+    classifiedAs: wd:Q4917 # US Dollar
+    currentPossessedQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 845
+    currentOwnedQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 845
+
+  - '@id': bankb:3be5259d-10f0-431c-9fec-9c0c15a461d3
+    '@type': EconomicResource
+    skos:note: Bob's account
+    classifiedAs: wd:Q4917 # US Dollar
+    currentPossessedQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 530
+    currentOwnedQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 530
+
+  # transfer of currency from alice to bob - first banka sends the transaction to the central bank account
+
+  - '@id': urn:uuid:0468e9b9-4d54-4eee-9d3f-bc76db11710a
+    '@type': EconomicEvent
+    action: transfer
+    transferOf:
+    - possession
+    resourceClassifiedAs: wd:Q4917 # US Dollar
+    resourceInventoriedAs: banka:b75d8f6a-e2df-4e52-b36d-1a22a66f4ead # alice's account
+    toResourceInventoriedAs: centralbank:f47064eb-7120-4b31-b882-770165901fe6 # some central bank account
```

Comment:
How would you do it if banka doesn't expose how it does clearing with bankb (central bank or whatever else), it just shares with Alice information that it decrements her account by 100 units and takes possession of those units.

Similar with bankb, it may only share with Bob information that it incremented his account by 100 units from Alice.

Reply:
This transfer wouldn't be visible to alice or bob, just between the banks and the central bank.  Alice already wrote a check or initiated a transfer in some way, and then she just waits.

Reply:
As I think of the bank stuff, the reality is that alice loaned banka the amount in her account, so really none of these transfers would involve alice or bob as agents, just their accounts.  The only thing with alice would be the money between her and banka, not shown in my examples.  Crypto would be the same in terms of the focus being on the accounts, but wouldn't have that weirdness of the banks actually 'owning' the money with a debt to the depositor.  (That's just a bank rip-off anyhow - but we should be able to support any kinds.)